temporal-sdk/src/test/java/io/temporal/workflow/shared/TestWorkflows.java

```diff
@@ -158,6 +158,16 @@
     void mySignal(String value);
   }
 
+  @WorkflowInterface
+  public interface ExampleWorkflow {
```

Comment:
If you created `TestWorkflows` class with an intention of reusing interfaces/implementations from here across tests - the names of the classes should be more descriptive. `ExampleWorkflow` is not a descriptive name, any interface in this class can be named this name. Please rename it to something like `WorkflowWithSignal`.
BTW, there is already an interface `TestSignaledWorkflow` in this class that looks completely appropriate for this use-case, so please consider reusing instead of making a new one.